function/eval/eval.py

```diff
@@ -0,0 +1,21 @@
+std_1, std_2, std_3, std_4 = ['test1'], ['test2'], ['test3'], ['test4']
+
+print('#######\teval function\t#######')
+for i in range(1, 5):
+	print(eval('std_'+str(i)))
+
+print('#######\texec function\t#######')
+for i in range(1, 5):
+	print(exec('std_'+str(i)))
```

Comment:
```
exec('print(std_{})'.format(i))
```

Reply:
https://windard.com/project/2015/12/10/Function-In-Python#eval

Reply:
awesome suggestion!!!
Boss Yang is SO Niubility